### ISSUE TITLE
As a user, I would like to see content on the find a forest page that makes it clear I am buying a Christmas tree permit.

### ISSUE THREAD

**Issue by bryanmcfadden** [nciinc/fs-permit-platform#257]
## Notes
This comes from the sprint 06 usability testing
[Sprint 06 UT Summary](https://github.com/nciinc/fs-permit-platform/wiki/Sprint-06-Christmas-tree-ePermit-research-results-January-3,-2018)

## Acceptance Criteria
- [ ]

## Definition of Done
- [ ] Code unit / e2e tested and comply with QWASP
- [ ] No new security vulnerabilities
- [ ] Internal team code review
- [ ] Documentation / readme.md / CircleCI updated
- [ ] Docker updated
- [ ] Compare finished design with mockup
- [ ] Usability tested

**Comment by bryanmcfadden** [comment]
Jane and I came up with the language for this design as a proposal, but it still could be used for this story (the paragraph).  Even though this story does not include a redesign, I thought iit might be helpful to attach the design here.

Note - the color scheme and states colored do not apply to the design. 
![sp_07_pro_01_01](https://user-images.githubusercontent.com/16689692/34890314-6a4ebeee-f79f-11e7-9bc5-bea58be5cdd4.png)

When a user clicks on a state, the popup would show the forests for that state. As an alternative (and on mobile devices) a plain html table can also be viewed to select a forest.

![sp_07_pro_01_02](https://user-images.githubusercontent.com/16689692/34890319-6de612f0-f79f-11e7-8638-63799c3435c4.png)

**Comment by sethalt** [comment]
@tram @MelissaBraxton Please review our revision:

<img width="1108" alt="screen shot 2018-01-16 at 9 37 16 am" src="https://user-images.githubusercontent.com/2347759/34997023-efa4b630-faa0-11e7-97ed-c58a7f9bd250.png">

**Comment by MelissaBraxton** [comment]
Suggest making it more clear that this is an entry point to online purchasing, so that we don't lead folks to believe this is an entry point for all Christmas tree permit purchasing information for all forests.  

 - Suggest changing the title to "Buy a Christmas tree permit online." 
 - Add content in the para below that tells people **how to find** forests that sell Christmas tree permits online.  Something like "Here you can find a forest that sells online Christmas tree permits near you"

**Comment by bryanmcfadden** [comment]
@MelissaBraxton - Changed title, added 2 lines for user instruction.

![image](https://user-images.githubusercontent.com/16689692/35010960-9030ee94-fad2-11e7-9091-6fc536249cfa.png)

**Comment by bryanmcfadden** [comment]
@MelissaBraxton @tram  - how about this?

![image](https://user-images.githubusercontent.com/16689692/35055908-531d0090-fb7e-11e7-99cb-543ae69c704c.png)

**Comment by tram** [comment]
👍 Ship it!

**Comment by sadlerw** [comment]
![image.png](https://images.zenhubusercontent.com/595e640361b4ca3b0a7370e7/83204765-6c1a-468d-b4fd-f8ed86801d1f)

**Comment by ASprinkle** [comment]
Remove "Not all forests offer Christmas tree cutting season"  And can we change the rest to read something like this? 
"For National Forests that offer a Christmas tree cutting season, you can buy a permit at the local Ranger District office.  During this Christmas tree cutting season, the Forests listed below are offering an opportunity to buy a Christmas tree permit online as part of a pilot project."  

Also the header for the Forests seems clunky as it is written "Pilot Christmas tree permit forests".  Can we change it to something like: "National Forests participating in the online Christmas tree pilot program"  But even that seems too long...

**Comment by sadlerw** [comment]
yea that's been addressed in another story it now looks like:

![Screen Shot 2018-01-18 at 1.23.06 PM.png](https://images.zenhubusercontent.com/595e640361b4ca3b0a7370e7/f2352ce3-348a-47ff-9eca-270dfd7a938c)

**Comment by JaneZC** [comment]
Here's some updated text ...

National Forests have their own rules, season dates and permit prices for cutting down Christmas trees.  During the Christmas tree cutting season, the Forests below offer the option to buy a Christmas tree permit online. For National Forests that do not offer the online permit option, you can buy a permit at the local Ranger District office.

**Comment by ASprinkle** [comment]
Great re-work @JaneZC  - i have to additions:
"During [this] Christmas tree cutting season..." 
and
..."the Forests below are offering the option to buy a Christmas tree permit online [as a pilot program]."  Shouldn't we include a "pilot" statement so that people don't assume this is a permanent thing. Since this is a pilot, there is a possibility that the option to buy online may not be available the following season.

**Comment by sethalt** [comment]
@ASprinkle I've updated the content per you and Jane's suggestions, please review and approve: 

![screen shot 2018-01-18 at 4 32 01 pm](https://user-images.githubusercontent.com/2347759/35124867-3c6d7e2a-fc6d-11e7-9a63-3b3f95dfe186.png)

**Comment by ASprinkle** [comment]
The content in that paragraph appears to be the same as in the previous screen shot.

**Comment by JaneZC** [comment]
@ASprinkle In terms of mentioning the pilot project ... I am not sure that folks will really put it all together just because we used the word 'pilot.' (Meaning, does the word pilot automatically tell the reader that this module could go away sometime in the future?) My default is always going to be to reduce wordiness ... and so left out the pilot forest mention, to reduce the text on the page and because I'm not sure it'll deliver the message we're hoping for. With that said ... I don't want to spend tons of time on it and, since you prefer it, I think we can add it back in. :)

An idea, though, that is outside the scope of this particular story  .... Do we have plans to collect feedback from users once this is rolled out? (Like using Survey Monkey or something?) If so, that would be a better way to work in the info about it being a pilot. For example - with a paragraph like: "We are currently piloting this online Christmas tree permit purchase process and would appreciate your feedback. Let us know what you think (by filling out survey, etc.) so we can make decisions to improve this service." You'd add the language to the email, the order page, and (probably) the landing page. Just a thought for future planning ...

**Comment by ASprinkle** [comment]
@JaneZC You're correct, people may not know what that means, however I would like to see if this comes up in @bryanmcfadden 's future usability testing.  If it's an issue we can remove "as a pilot program" and end then sentence.
As for roll out, you're right it is outside of the scope, and I do not have the inside track information on how roll out will occur. I would hope that the pilot discussion is part of the roll out.

**Comment by JaneZC** [comment]
@ASprinkle @bryanmcfadden I think it would be a great for Bryan to ask about ... ie. what information would people want to know about this being a pilot, etc. etc. What would they find helpful?

**Comment by tram** [comment]
Also, is it an issue if a visitor doesn't know that "pilot" = "test".

**Comment by ASprinkle** [comment]
@JaneZC @bryanmcfadden @tram 
Remove the "as a pilot program".  It appears that this may create more problems in the future.  Since this story doesn't mention dealing with clarity for the "pilot" program and instead specifies clarity on purchasing a Christmas tree permit let's just focus on the MVP for the story.  Remove the "as a pilot program" language and end the sentence.  and call it good from there.
